### PR TITLE
wfss unit tests

### PR DESCRIPTION
**Pull request by nden**
Prep work for refactoring the wfss transforms.

### CODE REVIEW

**Comment by nden** [comment]
The Jenkins error is unrelated and it passed the previous run. Opening and closing - hoping to restart Jenkins.

File: jwst/assign_wcs/tests/test_wfss.py [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+    # ymax = 58
+    # xmax = 1323
+    model = Mapping((0, 1, 0, 0, 0)) | (Shift(xmin) & Shift(ymin) &
+                                        Const1D(x0) & Const1D(y0) & Const1D(order) )| model
+    wcsobj = wcs.WCS([('det', model), ('world', None)])
+    # refactored call
+    # wcsobj.bounding_box = ((3, 7), (2, 5), (x0-1, x0+1), (y0-1, y0+1), (1, 3))
+
+    #wcsobj.bounding_box = ((xmin, xmax), (ymin, ymax))
+    wcsobj.bounding_box = ((20, 25), (800, 805))
```
Comment (hbushouse) [inline jwst/assign_wcs/tests/test_wfss.py]: These values don't match your values for xmin, xmax, ymin, ymax above. Is that OK?

File: jwst/assign_wcs/tests/test_wfss.py [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+                                                  wavelength[j, i], 1)
+
+    expected_xdx = np.array([[20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.]])
+
+    expected_ydy = np.array([[1584.5, 1584.50000003, 1584.5, 1584.5, 1584.5, 1584.5],
```
Comment (hbushouse) [inline jwst/assign_wcs/tests/test_wfss.py]: What level of precision does `assert_allclose` use? I'm wondering if you need to specify "1584.50000003" or would "1584.5" be good enough.

File: jwst/assign_wcs/tests/test_wfss.py [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+CRVAL2  =     -27.792691818741 / [deg] Coordinate value at reference point
+LONPOLE =                180.0 / [deg] Native longitude of celestial pole
+LATPOLE =     -27.792691818741 / [deg] Native latitude of celestial pole
+MJDREFI =                  0.0 / [d] MJD of fiducial time, integer part
+MJDREFF =                  0.0 / [d] MJD of fiducial time, fractional part
+RADESYS = 'ICRS'               / Equatorial coordinate system
+SPECSYS = 'BARYCENT'           / Reference frame of spectral coordinates
+    """
+    fits_wcs = wcs.WCS(fits.Header.fromstring(fits_header, sep='\n'))
+    fa = asdf.open('grism_wcs.asdf')
```
Comment (hbushouse) [inline jwst/assign_wcs/tests/test_wfss.py]: Where is "grism_wcs.asdf"?

File: jwst/assign_wcs/tests/test_wfss.py (hunk 1 above)
Comment (nden) [reply jwst/assign_wcs/tests/test_wfss.py]: I didn't want to compare big arrays just a few numbers.

File: jwst/assign_wcs/tests/test_wfss.py (hunk 3 above)
Comment (nden) [reply jwst/assign_wcs/tests/test_wfss.py]: Oops , forgot to commit it.

**Pull request merged**